app/src/main/java/org/kinecosystem/kinit/view/earn/WebTaskTruexFragment.kt

```diff
@@ -16,6 +18,16 @@ import org.kinecosystem.kinit.viewmodel.earn.WebFragmentActions
 import org.kinecosystem.kinit.viewmodel.earn.WebTaskTruexViewModel
 
 class WebTaskTruexFragment : BaseFragment(), WebFragmentActions {
+    override fun openBrowser(url: String) {
+        val parsedUrl = Uri.parse(url)
+        val intent = Intent(Intent.ACTION_VIEW, parsedUrl)
+        if (parsedUrl == null || intent.resolveActivity(context!!.packageManager) == null) {
```

Comment:
why crash here? context?.packageManager is better

Reply:
If `resolveActivity` receive a `null` it will crash as well